packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -61,10 +61,10 @@ export default {
       verboseEvents: true
     },
     resolution: {
-      maxWidth: 1280,
-      maxHeight: 720,
-      idealWidth: 1280,
-      idealHeight: 720
+      maxWidth: 1920,
+      maxHeight: 1080,
+      idealWidth: 1920,
+      idealHeight: 1080
```

Comment:
is this changing the defaults for everybody using JS-SDK to now try 1080p instead of 720p? Maybe we should let the backend media team know about this, just so that they expect the extra load.

Reply:
Reverted back to 720p as the default value for local video stream is 720p.

Reference - packages/node_modules/@webex/plugin-meetings/src/media/properties.js : line number28
```
    this.localQualityLevel = options.localQualityLevel || QUALITY_LEVELS['720p'];
    this.remoteQualityLevel = options.remoteQualityLevel || QUALITY_LEVELS.HIGH;
```